figures/generate-figures.sh

```diff
@@ -3,26 +3,39 @@
 # Run all figure making scripts. 
 
 # Find current directory based on this script
-cd "$(dirname "${BASH_SOURCE[0]}")" 
- 
-# Make an output folder for all directories
+cd "$(dirname "${BASH_SOURCE[0]}")"
+analyses_dir=../analyses
+
+# Make output folders for all figures
 mkdir -p pngs
+mkdir -p pdfs
 
 ## Sample distribution
 
 ################ Mutational landscape figure
 # Run both SNV caller consensus scripts
 # Note: This the PBTA consensus script requires at least 128 MB of RAM to run
-bash ../analyses/snv-callers/run_caller_consensus_analysis-pbta.sh
-bash ../analyses/snv-callers/run_caller_consensus_analysis-tcga.sh
+bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-pbta.sh
+bash ${analyses_dir}/snv-callers/run_caller_consensus_analysis-tcga.sh
 
 # Run mutational signatures analysis
 Rscript -e "rmarkdown::render('../analyses/mutational-signatures/mutational_signatures.Rmd', clean = TRUE)"
 
 # Run the figure assembly
 Rscript scripts/fig2-mutational-landscape.R
 
-## Interaction plots 
+######################
+## Interaction plots
+
+# Run the main figure generation script
+bash ${analyses_dir}/interaction-plots/01-create-interaction-plots.sh
+
+# Copy the main figure to final directory
+cp ${analyses_dir}/interaction-plots/plots/combined_top50.pdf pdfs/mutation_cooccurrence_figure.pdf
```

Comment:
For manubot purposes, I think we may want a PNG copy as well but not entirely sure. From the [Usage documentation about figures](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures):

> For images created by the manuscript authors that are hosted elsewhere on GitHub, we recommend using a [versioned](https://help.github.com/en/github/managing-files-in-a-repository/getting-permanent-links-to-files) GitHub URL to embed figures, thereby preserving exact image provenance. When embedding SVG images hosted on GitHub, it's necessary to append ?sanitize=true to the raw.githubusercontent.com URL.

We can move forward with the PDF and find out.

Reply:
I think I will just change back to PNG. Sad to lose resolution, but if manubot isn't going to support it, we will just have to live with it.